### PR TITLE
added Circular Queue

### PR DESCRIPTION
**Pull request by KashyapNasit**
Program to implement Circular Queue in C

### CODE REVIEW

**Comment by KashyapNasit** [comment]
Program to implement Circular Queue in C

File: data_structures/Circular Queue.c [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#include<stdio.h>
+#define sizeQueue 10
+int totalElement;
```
Comment (christianbender) [inline data_structures/Circular Queue.c]: Please assign this variable a useful start-value. For example 0

File: data_structures/Circular Queue.c [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+struct queue{
```
Comment (christianbender) [inline data_structures/Circular Queue.c]: Please format your full code according this code style conventions  
https://www.gnu.org/prep/standards/html_node/Formatting.html#Formatting
https://www.gnu.org/prep/standards/html_node/Syntactic-Conventions.html#Syntactic-Conventions
https://www.gnu.org/prep/standards/html_node/Writing-C.html

File: data_structures/Circular Queue.c [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+	int i,j=totalElement;
+	//printf("%d %d",q.start,q.end);
+	for(i = (q.start+1)%sizeQueue; j-- ; i = (i+1)%sizeQueue)
+		printf("%d\n",q.info[i]);
+}
+
+void main(){
+	char c;
+	int i;
+	Queue qu;
```
Comment (christianbender) [inline data_structures/Circular Queue.c]: Use dynamic memory managment for this data structure

File: data_structures/Circular Queue.c [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+			case '2':
+				i = dequeue(&qu);
+				printf("%d is dequeued.\n",i);
+				break;
+			case '3':
+				display(qu);
+				break;
+		}
+		printf("\n\n");
+	}while(1);
```
Comment (christianbender) [inline data_structures/Circular Queue.c]: Put in a ```return 0;``` at the end of the main-function

**Pull request closed without merging**